schema/gen/go/genKindLinkNode.go

```diff
@@ -98,6 +98,16 @@ func (gk generateKindLink) EmitTypedNodeMethodRepresentation(w io.Writer) {
 	`, w, gk)
 }
 
+func (gk generateKindLink) EmitTypedLinkNodeMethodReferencedLinkBuilder(w io.Writer) {
```

Comment:
s/ReferencedLinkBuilder/ReferencedNodeBuilder/ ?

Reply:
awesome, done, and then I'm gonna merge